src/mkdocs_include_markdown_plugin/event.py

```diff
@@ -2,6 +2,7 @@
 
 from __future__ import annotations
 
+import copy
```

Comment:
`copy` is not used. Please, remove it.

And read the [*CONTRIBUTING.md* file](https://github.com/mondeja/mkdocs-include-markdown-plugin/blob/master/CONTRIBUTING.md) to know how to run the tooling in your local environment, that would speed the reviewing process.

Reply:
I'm sorry for the inconvenience. I've actually followed the guidelines and used `hatch`, now